README.md

```diff
@@ -114,3 +114,45 @@ The interesting fields for this use case are:
   the SSO user ID to be sent to Segment
 - `status.compliantUsername` - Contains the username used in the cluster audit
   log.
+
+## Setting up a containerized Splunk instance
+- Either podman or docker can be user
+- running the following command will deploy a Splunk enterprise container:  
+- `podman pull docker.io/splunk/splunk:latest`
+  `podman run -p 8000:8000 -e "SPLUNK_PASSWORD=AcceptablePassword1" -e "SPLUNK_START_ARGS=--accept-license" -it --name splunk_container docker.io/splunk/splunk:latest`
+- More info about the specific inputs of the command can be found [here|https://docs.splunk.com/Documentation/Splunk/9.0.4/Installation/DeployandrunSplunkEnterpriseinsideDockercontainers]
+
+
+## Setting up a containerized Splunk instance
+
+To set up a containerized Splunk instance, you can use either podman or docker. Follow these steps:
+
+1. Pull the latest Splunk Enterprise container image:
+   ```
+   podman pull docker.io/splunk/splunk:latest
+   ```
+
+2. Run the container with the desired configuration:
+   ```
+   podman run -p 8000:8000 -e "SPLUNK_PASSWORD=YourPassword" -e "SPLUNK_START_ARGS=--accept-license" -it --name splunk_container docker.io/splunk/splunk:latest
+   ```
+   This command maps port 8000 from the container to the host machine, sets the Splunk password to meet the required complexity, and accepts the Splunk license.
```

Comment:
What is the required complexity? Don't make me have to follow a link to know that!

Reply:
changed